i18n/src/.eslintrc.json

```diff
@@ -0,0 +1,6 @@
+{
+  "env": {
+    "browser": false,
```

Comment:
>
> In i18n/src/.eslintrc.json
> <https://github.com/angular/angular.js/pull/14952#discussion_r72910207>:
>
> > @@ -0,0 +1,6 @@
> > +{
> > +  "env": {
> > +    "browser": false,
>
> Do we need "browser": false is all .eslintrc.json files?
>
Technically, no (and I'm afraid I'm not being consistent in this PR in this
regard); it's just ESLint by default extends rules up the tree and we have
code structure nested in funny ways so sometimes we have to overwrite
`browser: true` up the tree.

Maybe it'd be better to stop relying on the default inheritance and use
extends as we did with JSHint? Then we'd only have rules in the extended
files and the rest (env, globals) would be declared in all the files.

Thoughts? I can change the files like that if you think it'd be better.